downstairs/src/lib.rs

```diff
@@ -466,6 +466,37 @@ pub mod cdt {
     fn extent__read__file__done(job_id: u64, extent_id: u32, n_blocks: u64) {}
 }
 
+// Check if a Message is valid on this downstairs or not.
```

Comment:
No change here, I just made this code a function and had everyone call it, instead of
copying the same code for over half of the matches.